gensim/models/keyedvectors.py

```diff
@@ -818,10 +825,13 @@ def most_similar_cosmul(self, positive=None, negative=None, topn=10):
 
         self.init_sims()
 
-        if isinstance(positive, string_types) and not negative:
+        if isinstance(positive, string_types):
             # allow calls like most_similar_cosmul('dog'), as a shorthand for most_similar_cosmul(['dog'])
             positive = [positive]
 
+        if isinstance(negative, string_types):
```

Comment:
If I understand correctly, this enables behavior like:

    most_similar_cosmul('dog', 'cat')

where `dog` is positive and `cat` is negative. That's helpful shorthand, but without documentation, people won't find out about it.

Can you please add a paragraph to the docstring explaining the above shorthand?

Reply:
Note that this just makes the special type-testing treatment of `negative` match that of `positive` (in both `most_similar()` and `most_similar_cosmul()`) – but that special treatment, while used extensively in examples, isn't currently documented even in the `most_similar()` case! I'd suggest that treating `negative` symmetrically with `positive` is a good idea, and should also be done in `most_similar()` for consistency, and both of their doc-comments should be improved/harmonized to explain this behavior.

Reply:
@mpenkov added shorthand in docstring for clarity, is it okay or is there some other place for docs as well that this should be added?